pkg/lib/machines.js

```diff
@@ -593,6 +620,7 @@
                         open = true;
                         if (url)
```

Comment:
It feels a little weird to get the manifest like this instead of getting from the dbus. I know we do it so that we can ensure we have the etag for subsequent requests. So unless we come up with a better way to do that I guess this is ok.